test/edu/ithaca/dragonlab/ckc/conceptgraph/ConceptNodeTest.java

```diff
@@ -1,12 +1,71 @@
 package edu.ithaca.dragonlab.ckc.conceptgraph;
 
 import edu.ithaca.dragonlab.ckc.conceptgraph.ConceptNode;
+import edu.ithaca.dragonlab.ckc.io.LinkRecord;
+import edu.ithaca.dragonlab.ckc.learningobject.ExampleLearningObjectFactory;
 import org.junit.Assert;
 import org.junit.Test;
 
+import java.util.HashMap;
+
 public class ConceptNodeTest {
 
-	@Test
+    @Test
+    public void isAncestorTest() {
+        ConceptGraph orig = PracticalConceptGraphTest.simpleTestGraphTest();
+
+        ConceptNode node = orig.findNodeById("C");
+        boolean num = orig.findNodeById("A").isAncestorOf(node);
```

Comment:
IsAncestorOf doesn't return a boolean, right?
Also, num is not a good variable name for a boolean